processing/2_validate_compendia.ipynb

```diff
@@ -0,0 +1,2851 @@
IiIjI5zEpJiIiIiKfx6SYiIiIiHwek2IiIiIi8nlMiomIiIjI5zEpJiIiIiKfx6SYiIiIiHwek2IiIiIi8nlMiomIiIjI5zEpJiIiIiKfp/N2AERE9OeWb7fjRH4h7CKI8TOinkHv7ZCIiKock2IiIirTqfxCLPktFRlWGzQAogw6pNSvh+ZBAd4OjYioSjEpLiUgIAA6nQ4iUiPLExF1eTW1zKrijLe2xQ2w3r2F9e4d11LvVrsdL546j4t2OwwKYAVwusiGDy5kYI7ZD2attnqC/p2v1ru31bZ61+mYylDV4J5USlJSEkJDQ2Gz2WpsmaGhoXA4HHA4HDW2zKpkt9u9HUKlsN69g/XuHZWt94OWfGTY7TAA8NNoIA5BPgRnrTZkFxfDoK+ZZhS+Vu9/FrWl3kNDQ70dAtURTIpL2b9/P1q1aoWIiIgaWZ7D4UBGRgbCw8Oh0dSudx5FBHa7HVqtFoqieDucq8J69w7Wu3dUtt4dIki32uFM5xwANBoFsAsUAAF6fbXfofNGvTuKimC35EKBAm1QEJRKJv7c32tOWlqat0OgOoJJcSkWiwU2m63GDgKKoqjLqw0HHk9qY+ysd+9gvXtHZeq9yG7HBxcysCsnF3aUNJuw2R3QABAAzcwmBNXQXWKg5urdnpMDy8F9KDp1EnZLHnThYQjp3R/6sPCrnhf395pTk093qW5jUkxERCoRwfLfLmJnbh5KPzx3oKQPz2YmI2bG1szTtJokIsg7/DPyfzoI64ULsFsssB/Ig2X/PsTeOQuGBnHeDpGIqlnteqZDRETV6rClALt+T4iV3/8AwASgU5A/7oyrD1MN3iWuKWK1wnr2DKwXLsCanQV7ViZQXAj7ubP4bclCWDPTvR0iEVUzJsVERKT6IccCG0qS4dIniGIAkQY9/Ovom/6KTgcRB+wWC8SSC2i0gN4AGPRwZGYia+vn3g6RiKoZk2IiIlIJAGdHa5f3PRBrMMCvlr00VlGKRgNT0xYQux2w2X6/RS6ARgNotSg8cxqOWtIbAxFVTt08uhERUaU0CzDDrNGoTSecPdW28DPArtSevmsrw79lK5iTbgQUDfB7ApyvN+KiyR/nc3KQum4NrDk5Xo6SiKpL3XwORkRElZIU5I/+hUHYnJYFGwCtAjT2M+CmkCBvh1YjIlPGw3EpEwVHDuFsYDB+uu4GZIaGwaHTon5mBm5a/iqaz7wbepPJ26ESURVjUkxEPsWelwdHngWi0UAfEgqljraRrSyNomBYZDjq6fU4ml+IegYdwnV65DocuMFoqBVddF0LjU6HqBl/xamvtuHIhYvI8fNHcF4uFI0G6cEh2FtchNAtGxE3eBiUOtqUhMhXXdXZYOvWrVi1ahXOnTuHqKgoDBkyBIMHDy6z/L/+9S989tln2LZt2zUHSkR0razpaUj/ZgeOX0iFxe6A2c+M1n37Ijgu3tuh/aloFAVdQ4IQZTDgfHEx7BA0NfshwWT0dmg1QuPnB13Xnij+bAv0xcVwrrXRbkeByYTs8+cRm58HbUCgV+MkoqpV4aT4vvvuw8KFCwH80abs7bffRufOnfHuu+8iISHBbZr//e9/2L59exWFSkRUeY7iYmR+uxt7LlzEKf8QWPxMsEHBoR3fYERPLcJjYr0d4p+KTqOgWYAJTcQPQEmi7EsMOi00JjPsVhscDgcUpeQrf1qNpuRlHEfdbVtN5Ksq9Ozn008/xYIFC6DVavGXv/wFS5YswZw5c1CvXj3s2rULHTp0wPfff1/dsRIRVZoUFyE1LQ2nTAEoMJoQaLMhpKgAqf6B+Hr/QTjq8Atk10KjKD6XEANAuNGA66MiYfcPQHpIKNIDQ6BotYjNvoSIBrHQmM3eDpGIqliF7hS//PLLUBQFH3/8MW699VZ1+D/+8Q/MnDkTa9euRZ8+fbB+/Xr06dOn2oIlIqosRa+H3SEoNBlhEDv0DjvgcMBcXIgMxYhCh8Cs9b3kjzzTKQq6NW2KkIJC/PLbb3DkWxCTm4OmsdEI69WXbdGJ6qAK/ar37NmDDh06uCTEABAcHIzVq1cjKSkJ8+bNw8CBA7Fq1SoMGTKkWoIlIqosjdEPYY0bA2cvwqoAJpsNVo0WUDQwGYzw1XTYYrMj02qDRgFCdDoE6LRXnshHGP390b5TR7Rp3gyOwkJozP7QBQXV+ZcNiXxVhZpPZGdn47rrritz/IMPPoi33noLdrsdo0aNwjvvvFNlARIRVZWEbj2Q5KdDod4P6f6ByPfzQ6BOhxuuvx5+Gt9LdA5b8vHKbxew/FwaVqVm4MtLOciw2rwd1p+KotNBHxoGY3QM9MHBTIiJ6rAKJcXh4eFITU0tt8z48eOxbt06aLVaTJkyBUuWLKmSAImIqoqiKOg1eAgGN07ADWEhaBpVH13aJaFVw0SfS3ZSi4rxQVomMqxWGDQKcux2/GjJw4HcPLavJqojEhOr5ti2YsUKKIqC+fPnX3tQ5Th58iQURUHPnj2rdTllqVDziTZt2uDrr79GXl4e/P39yyw3cOBAbNmyBYMGDcKsWbMQERFRZYESEVUFjUaDVq1aoxVKetLxtWTY6VhBIfJsdtTT66HXKDBrNUgrtiK12AqbCAw+Wi9E5LsqdKc4OTkZhYWFePvtt69YtkePHvjyyy8RHh6OixcvXnOARETVxVcTYgBwOAR6RQNrqbvCdil5wUznw/VCRL6rQknx4MGDcfPNN2PHjh0Vmmm7du2wY8cOtGnTBvHx7BSfiOjPJtpoRLBOh2KHAzk2Oy4WW2HUKOgQaPbJLtiIiCqUFMfHx2Pnzp147733Kjzjpk2bYv/+/Thx4kSlgyMiourRyGxEr7BARBj0MCgKog16jIoMR0OzyduhEdUau3fvxuDBgxEREQGj0YjExET89a9/xblz59zKzp8/H4qiYMWKFR7n5an971dffQVFUTB58mSkpqZi+vTpaNCgAXQ6HRYtWlTpuDdu3IipU6fihhtuQFBQEPz9/dGmTRs
```

Comment:
Plots look nice! Is there some commonality between the clustered samples? For example, are they from the same experiment?

Reply:
Another great point! My thought was that strain type would be the largest source of variance in the data and so I would have expected clustering based on strain type. But this is not what you see, so looking into other factors like experiment type would be helpful to understand the data more

Reply:
There are a lot of study ids so its a little hard to see what is happening, but overall it looks like there is clsutering based on study id. I'm still not clear on why there is not more of a separation in the expression data for strain type, but perhaps this will become more clear later.

![image](https://user-images.githubusercontent.com/13107360/113608158-2dff0000-9618-11eb-8b62-5fa86c680f11.png)

![image](https://user-images.githubusercontent.com/13107360/113608170-32c3b400-9618-11eb-8b49-b70b718ce476.png)